git-extra/git-process.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
Check for an update, I am assuming this would be done from the installer.iss but I did not see any bash script being called from there.

Reply:
No, `installer.iss` defines what is run during the installer. Why would you check for an update when running the installer?